diff -Naur opencv/modules/core/include/opencv2/core/private.hpp opencv_fix/modules/core/include/opencv2/core/private.hpp
--- opencv/modules/core/include/opencv2/core/private.hpp	2019-11-01 16:11:51.973405764 +0700
+++ opencv_fix/modules/core/include/opencv2/core/private.hpp	2019-11-01 16:13:34.854675282 +0700
@@ -67,7 +67,7 @@
 #    pragma warning(disable:4702)  // unreachable code
 #    pragma warning(disable:4714)  // const marked as __forceinline not inlined
 #  endif
-#  include <Eigen/Core>
+#  include <eigen3/Eigen/Core>
 #  if defined(_MSC_VER)
 #    pragma warning(pop)
 #  endif
